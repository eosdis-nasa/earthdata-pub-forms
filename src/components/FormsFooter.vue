<template>
  <footer class="mt-auto">
    <div class="container">
      <nav role="navigation">
        <ul class="footer-links">
          <li>NASA Official: Justin Rice</li>
          <li><a href="https://www.nasa.gov/FOIA/index.html" target="_blank" alt="FOIA" title="FOIA">FOIA</a></li>
          <li><a href="https://www.nasa.gov/about/highlights/HP_Privacy.html" target="_blank" alt="NASA Privacy Policy" title="NASA Privacy Policy">NASA Privacy Policy</a></li>
          <li><a href="https://www.usa.gov/" target="_blank" alt="USA dot gov" title="USA dot gov">USA.gov</a></li>
          <li><a href="javascript:feedback.showForm();" alt="Feedback Form" title="Feedback Form">Feedback</a></li>
          <!-- <li style="float: right"><span class="version badge eui-badge--md">v 1.193.3-r22.1.3</span></li> -->
        </ul>
      </nav>
    </div>
  </footer>
</template>
