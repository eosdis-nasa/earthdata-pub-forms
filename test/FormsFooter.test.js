import { mount} from "@vue/test-utils";
import FormsFooter from "@/components/FormsFooter"

describe("FormsFooter", () => {
    it('renders correctly', () => {
        const wrapper = mount(FormsFooter)
        expect(wrapper.element).toMatchSnapshot()
      })
})

