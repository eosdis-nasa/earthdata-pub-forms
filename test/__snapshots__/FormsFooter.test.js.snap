// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormsFooter renders correctly 1`] = `
<footer
  class="mt-auto"
>
  <div
    class="container"
  >
    <nav
      role="navigation"
    >
      <ul
        class="footer-links"
      >
        <li>
          NASA Official: Doug Newman
        </li>
         
        <li>
          <a
            alt="FOIA"
            href="https://www.nasa.gov/FOIA/index.html"
            target="_blank"
            title="FOIA"
          >
            FOIA
          </a>
        </li>
         
        <li>
          <a
            alt="NASA Privacy Policy"
            href="https://www.nasa.gov/about/highlights/HP_Privacy.html"
            target="_blank"
            title="NASA Privacy Policy"
          >
            NASA Privacy Policy
          </a>
        </li>
         
        <li>
          <a
            alt="USA dot gov"
            href="https://www.usa.gov/"
            target="_blank"
            title="USA dot gov"
          >
            USA.gov
          </a>
        </li>
         
        <li>
          <a
            alt="Feedback Form"
            href="javascript:feedback.showForm();"
            title="Feedback Form"
          >
            Feedback
          </a>
        </li>
      </ul>
    </nav>
  </div>
</footer>
`;
